central/metadata/service/service_impl_test.go

```diff
@@ -224,6 +225,7 @@ func (s *serviceImplTestSuite) TestGetCentralCapabilities() {
 			s.Equal(v1.CentralServicesCapabilities_CapabilityAvailable, caps.CentralScanningCanUseContainerIamRoleForEcr)
 			s.Equal(v1.CentralServicesCapabilities_CapabilityAvailable, caps.CentralCanUseCloudBackupIntegrations)
 			s.Equal(v1.CentralServicesCapabilities_CapabilityAvailable, caps.CentralCanDisplayDeclarativeConfigHealth)
+			s.Equal(v1.CentralServicesCapabilities_CapabilityAvailable, caps.GetCentralCanUpdateCert)
```

Comment:
```suggestion
			s.Equal(v1.CentralServicesCapabilities_CapabilityAvailable, caps.GetCentralCanUpdateCert())
```

or

```suggestion
			s.Equal(v1.CentralServicesCapabilities_CapabilityAvailable, caps.CentralCanUpdateCert)
```

(the latter is what rest of the block do)

Reply:
I chose the latter suggestion, to stay consistent with the current pattern in the second test.